test/parallel/test-diagnostics-channel-memory-leak.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+const cp = require('child_process');
+
+const code = `
+  const { subscribe, unsubscribe } = require('diagnostics_channel');
+  function noop() {}
+
+  const before = process.memoryUsage().heapUsed;
+
+  for (let i = 0; i < 100; i++) {
+      subscribe(String(i), noop);
+      unsubscribe(String(i), noop);
+  }
+
+  gc();
+
+  const after = process.memoryUsage().heapUsed;
+  assert.ok(before >= after);
+`;
+
+const proc = cp.spawnSync(process.execPath, ['--expose-gc', '-e', code ]);
```

Comment:
Respawning the process with the flags is already done by `test/common/index.js` in https://github.com/nodejs/node/blob/d09f0c4dbc28acd89519064cec0b02d51842d985/test/common/index.js#L81, so this doesn't need to be done manually. Adding the `// Flags: --expose-gc` comment to the test file and doing `require('../common');` should be enough, right?

Reply:
Done, i have copy your code into test file, Thanks.